src/transformStateWithClones.js

```diff
@@ -8,6 +8,46 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const stateClonesArr = [];
+  let stateClone = { ...state };
+
+  function removeProps(propsToRemove) {
+    for (const key of propsToRemove) {
+      stateClone = { ...stateClone };
```

Comment:
Why do you do it?

Reply:
i am still a bit confused with this cloning and reference thing with clones have maybe different material i can read to understand it better ?  i  placed stateClone to different spot is that what you meant ?